test/madetomeasure_api/test/routes/subscribers.clj

```diff
@@ -11,6 +11,14 @@
     (body body-string)
     (content-type "application/json")))
 
+(defn json-delete [endpoint]
```

Comment:
Thinking this is starting to smell like a test support feature.

`json-delete, json-get, json-post, json-patch`